src/style.css

```diff
@@ -0,0 +1,31 @@
+.body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  width: fit-content;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
```

Comment:
last star shouldn't have margin. use `:not(:last-child)`

Reply:
@Viktor-Kost didn't get it. where exactly should I use this? I tried and did not succeed